engine/src/flutter/shell/platform/linux/fl_engine_private.h

```diff
@@ -96,7 +96,7 @@ FlDisplayMonitor* fl_engine_get_display_monitor(FlEngine* engine);
  * fl_engine_start:
  * @engine: an #FlEngine.
  * @error: (allow-none): #GError location to store the error occurring, or %NULL
- * to ignore.
+ * to ignore. If `error` is not %NULL, `*error` must not be uninitialized.
```

Comment:
This is a double negative and thus hard to understand - suggest instead "If provided, `*error` must be initialized.".

Reply:
Okay, this should be much easier to understand:

```
/**
 * fl_engine_start:
 * @engine: an #FlEngine.
 * @error: (allow-none): #GError location to store the error occurring, or %NULL
 * to ignore. If `error` is not %NULL, `*error` must be initialized (typically
 * %NULL, but an error from a previous call using GLib error handling is explicitly
 * valid).
 *
```

Is it good? :slightly_smiling_face:

Silly me, I jumped the gun and committed this change to one file. :facepalm:

Reply:
Maybe "using GLib error handling" is unnecessary...

Reply:
Looks good. I think it's better to err on the side of being more verbose since consumers are less likely to understand the GLib conventions.